src/main/resources/google_checks.xml

```diff
@@ -48,12 +48,18 @@
 
     <module name="TreeWalker">
         <module name="OuterTypeFilename"/>
+        <module name="WhitespaceAfter">
+            <property name="tokens"
+            value="COMMA, SEMI, TYPECAST, LITERAL_IF, LITERAL_ELSE,
+            LITERAL_WHILE, LITERAL_DO, LITERAL_FOR, DO_WHILE"/>
+        </module>
         <module name="IllegalTokenText">
             <property name="tokens" value="STRING_LITERAL, CHAR_LITERAL"/>
             <property name="format"
              value="\\u00(09|0(a|A)|0(c|C)|0(d|D)|22|27|5(C|c))|\\(0(10|11|12|14|15|42|47)|134)"/>
             <property name="message"
-             value="Consider using special escape sequence instead of octal value or Unicode escaped value."/>
+             value="Consider using special escape sequence instead of octal value
```

Comment:
Please revert this change. The violation should be suppressed in https://github.com/checkstyle/checkstyle/blob/aa5a9fd19a6c2e05dbfa756e259f7f59b7d676ce/config/suppressions.xml#L9

Reply:
Done